app/com/linkedin/drelephant/spark/fetchers/SparkRestClient.scala

```diff
@@ -83,17 +82,19 @@ class SparkRestClient(sparkConf: SparkConf) {
     val (applicationInfo, attemptTarget) = getApplicationMetaData(appId)
     val jobData = getJobDatas(attemptTarget)
     Future {
+      val appConfigurationProperties = if (fetchLogs) {
+        Future {
+          Option(getSparkConfigs(attemptTarget))
+        }
+      } else Future.successful(None: Option[ApplicationConfigImpl])
+      
       val futureStageDatas = Future {
         getStageDatas(attemptTarget)
       }
       val futureExecutorSummaries = Future {
         getExecutorSummaries(attemptTarget)
       }
-      val futureLogData = if (fetchLogs) {
-        Future {
-          getLogData(attemptTarget)
-        }
-      } else Future.successful(None)
+      val futureLogData = Future.successful(None)
```

Comment:
Please see if we can remove futureLogData , since its value is none

Reply:
Removed.